pkg/serving/config_changes.go

```diff
@@ -42,6 +51,36 @@ func UpdateEnvVars(template *servingv1alpha1.RevisionTemplateSpec, toUpdate map[
 	return nil
 }
 
+// UpdateEnvFrom updates envFrom
+func UpdateEnvFrom(template *servingv1alpha1.RevisionTemplateSpec, toUpdate []string, toRemove []string) error {
+	container, err := ContainerOfRevisionTemplate(template)
+	if err != nil {
+		return err
+	}
+	envFrom, err := updateEnvFrom(container.EnvFrom, toUpdate)
+	if err != nil {
+		return err
+	}
+	container.EnvFrom, err = removeEnvFrom(envFrom, toRemove)
+	return err
+}
+
+// UpdateVolumeMounts updates the configuration for mounting with config maps or secrets.
+func UpdateVolumeMounts(template *servingv1alpha1.RevisionTemplateSpec, toUpdate map[string]string, toRemove []string) error {
+	container, err := ContainerOfRevisionTemplate(template)
+	if err != nil {
+		return err
+	}
+
+	volumes, volumeMounts, err := updateVolumeMountsFromMap(template.Spec.Volumes, container.VolumeMounts, toUpdate)
+	if err != nil {
+		return err
+	}
+
+	template.Spec.Volumes, container.VolumeMounts = removeVolumeMounts(volumes, volumeMounts, toRemove)
+	return nil
+}
```

Comment:
Can we sort 'em too?

Reply:
Could you explain which parts should be sorted in more details?

Reply:
This part also seems gone while splitting the flags.